packages/core/chrome/core-chrome-browser-internal/src/ui/header/screen_reader_a11y.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useState, useEffect } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { EuiScreenReaderLive } from '@elastic/eui';
+
+import type { HeaderProps } from './header';
+
+const DEFAULT_TITLE = 'Elastic'; // This may need to be DRYed out with https://github.com/elastic/kibana/blob/main/packages/core/rendering/core-rendering-server-internal/src/views/template.tsx#L34
+const SEPARATOR = ' - ';
+
+export const ScreenReaderRouteAnnouncements: FC<{
+  breadcrumbs$: HeaderProps['breadcrumbs$'];
+}> = ({ breadcrumbs$ }) => {
+  const [routeTitle, setRouteTitle] = useState('');
+  const fallbackBreadcrumbs = useObservable(breadcrumbs$, []);
```

Comment:
Shoot sorry, I just realized I left this var name in as `fallbackBreadcrumbs` back from when my code was still trying to observe the `document.title` and use that instead of breadcrumbs (when breadcrumbs turned out to be the better & more complete titling method).

Please feel free to rename this across the file to just `breadcrumbs`, e.g.
```suggestion
  // We use breadcrumbs instead of `document.title` to obtain the page title as unfortunately
  // many Kibana plugins forget to set a page title, but almost all set comprehensive breadcrumbs
  const breadcrumbs = useObservable(breadcrumbs$, []);
```